.sfdevrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "exclude-scripts": ["prepack"]
```

Comment:
What does this do? The other PRs have not had this

Reply:
This plugin is on an old version of dev-scripts, which means that `oclif lock` is not part of the prepack script. I honestly didn't want to put in the effort to update to the latest dev-scripts so I decided to make the `prepack` script be `sf-prepack && oclif lock" and then exclude `prepack` from `dev-scripts`'s management

Reply:
Ahh that makes sense, thanks